src/platforms/php/guides/symfony/index.mdx

```diff
@@ -17,32 +17,106 @@ Install the `sentry/sentry-symfony` package:
 composer require sentry/sentry-symfony
 ```
 
-### Configure
+<Note>
 
-Add your DSN to `app/config/config.yml`:
+Due to a bug in all versions below `6.0` of the [`SensioFrameworkExtraBundle`](https://github.com/sensiolabs/SensioFrameworkExtraBundle) bundle, you will likely receive an error during the execution of the command above related to the missing `Nyholm\Psr7\Factory\Psr17Factory` class. To workaround the issue, if you are not using the PSR-7 bridge, please change the configuration of that bundle as follows:
 
-```yaml {filename:app/config/config.yml}
+```yaml
+sensio_framework_extra:
+   psr_message:
+      enabled: false
+```
+
+For more details about the issue see [https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/710](https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/710).
+
+</Note>
+
+## Configure
+
+Add your DSN to `config/packages/sentry.yaml`:
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
+    dsn: "___PUBLIC_DSN___"
+```
+
+If you **are not** using Symfony Flex, you'll also need to enable the bundle in `config/bundles.php`:
+
+```php {filename:config/bundles.php}
+<?php
+
+return [
+    // ...
+    Sentry\SentryBundle\SentryBundle::class => ['all' => true],
+];
+```
+
+### Monolog Integration
+
+If you are using [Monolog](https://github.com/Seldaek/monolog), you also need these additional configuration and services to log the errors correctly:
+
+```yaml {filename:config/packages/sentry.yaml}
 sentry:
-  dsn: "___PUBLIC_DSN___"
+    register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
+
+monolog:
+    handlers:
+        sentry:
+            type: service
+            id: Sentry\Monolog\Handler
```

Comment:
Instead of using `type: service` why not use the [built-in `sentry` type](https://github.com/symfony/monolog-bundle/pull/313/files#diff-3ee12206c3b122be4593dbfdb19397c16b3d337f759fad3bde6c785d6f1261c0R671) and pass the `ClientInterface` as the `client_id`. This would remove the need to register the `Sentry\Monolog\Handler` manually:

```yaml
sentry:
    type: sentry
    client_id: Sentry\ClientInterface
```

Reply:
That builtin type refers to the SDK 2.0, we're on 3.0. It may still work, though.

Reply:
@Jean85 If it's any help, I've been using this configuration with `sentry/sentry` version 3 for a while and haven't experienced any problems.

Reply:
The built-in type is creating its own instance of the hub on the DI container and does not set it as current hub, so all integrations won't work as expected. It should definitely be possible to improve the Monolog bundle to make things work in the correct way, but as of now our way of instantiating the handler is safer